.circleci/config.yml

```
@@ -680,6 +680,21 @@ jobs:
           command: ./scripts/pylint_all.py
       - gitter_notify_failure_unless_pr
 
+  chk_mypy:
```

```suggestion
  chk_python_types:
```
Given that this name will be shown up in the UI, it should be probably slightly more meaningful to those not knowing about "mypy" but trying to avoid too long names (sadly), I think that the above proposal is a good sweet spot. What do you think?